examples/purchase-tester/src/main/java/com/revenuecat/purchasetester/OverviewViewModel.kt

```diff
@@ -80,6 +85,20 @@ class OverviewViewModel(private val interactionHandler: OverviewInteractionHandl
         }
     }
 
+    fun retrieveCustomerInfo() {
+        viewModelScope.launch {
+            runCatching {
+                customerInfo.value = Purchases.sharedInstance.awaitCustomerInfo()
+                Log.i("PurchaseTester", "Get Customer info returned Customer info: ${customerInfo.value}")
+            }.onFailure {
```

Comment:
I was wondering whether doing this or just do a try-catch within the coroutine... I kinda prefer the try-catch since it better narrows the origin of the exception? I think this is ok too though.

Reply:
I agree, replaced it with just catching the specific exception